### ISSUE TITLE
Numeric role with a specific snippet

### ISSUE THREAD

**Issue by juanjux** [bblfsh/client-python#21]
With newlines included:

```python


class Repo2nBOW(Repo2Base):
    @property
    def id2vec(self):
        return self._id2vec













```

This produce two instances of role `141` but only with the client-python, it works on the integration tests and other ways to produce the UAST, so probably the bug is in this project.

**Comment by juanjux** [comment]
The .proto files of the client-python were outdated. Will be fixed soon.

**Comment by juanjux** [comment]
The main problem is fixed by updating the protobuf files but the unittests are failing because of some position problems with the static method argument that must be fixed in pydetector.

**Comment by juanjux** [comment]
Should be fixed in v0.0.5, thanks for the report!
